public/locations-list-map.js

```diff
@@ -148,6 +150,23 @@ const addDonationSites = () => {
   }
 };
 
+const translatedFilterItems = () => {
+  const translated = {};
+  const filterItemKeys = Object.keys(FILTER_ITEMS);
+
+  for (let i = 0; i < filterItemKeys.length; i++) {
```

Comment:
nit: `for (const [filterItemKey, filterItem] of Object.entries(FILTER_ITEMS)` is the same thing